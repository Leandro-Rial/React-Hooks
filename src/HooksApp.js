import React from 'react'

const HooksApp = () => {
    return (
        <div>
            Helouda
        </div>
    )
}

export default HooksApp
