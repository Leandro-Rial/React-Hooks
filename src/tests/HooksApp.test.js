import React from 'react'
import { shallow } from "enzyme";
import HooksApp from "../HooksApp";

describe('Pruebas HooksApp', () => {
    
    let wrapper = shallow(<HooksApp />)
    
    beforeEach(() => {
        wrapper = shallow(<HooksApp />)
    })

    test('debe de mostar <HooksApp /> correctamente', () => {
        
        expect(wrapper).toMatchSnapshot()

    });

});