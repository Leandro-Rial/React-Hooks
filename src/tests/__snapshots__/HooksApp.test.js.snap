// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas HooksApp debe de mostar <HooksApp /> correctamente 1`] = `
<div>
  Helouda
</div>
`;
